bin/cli.js

```diff
@@ -12,12 +12,14 @@ if (userArgs.indexOf('-h') !== -1 || userArgs.indexOf('--help') !== -1) {
   // eslint-disable-next-line
   console.log('usage: pkcs7');
   // eslint-disable-next-line
-  return console.log('pkcs7 expects input on stdin and outputs to stdout');
+  console.log('pkcs7 expects input on stdin and outputs to stdout');
+  process.exit();
```

Comment:
The linter complained about using return here, I think process.exit makes more sense. Same for the code below this.